nilearn/plotting/tests/test_matrix_plotting.py

```diff
@@ -89,8 +89,7 @@ def test_sanitize_reorder(reorder):
 def test_sanitize_reorder_error(reorder):
     from ..matrix_plotting import _sanitize_reorder
     with pytest.raises(ValueError,
-                       match=("Parameter reorder needs to be "
-                              f"one of {VALID_REORDER_VALUES}")):
+                       match=("Parameter reorder needs to be ")):
```

Comment:
My dirty fix is to make the regex less strict.

Reply:
Yea I think that's ok. But do VALID_REORDER_ARGS and VALID_REORDER_VALUES need to be an unordered set? Can we just use a list instead?

Reply:
I thought I had tried in the first commit but now realizing that the square brackets would turn add some "grouping" to the regex...

``
FAILED nilearn/plotting/tests/test_matrix_plotting.py::test_sanitize_reorder_error[None] - AssertionError: Regex pattern did not match.
 Regex: "Parameter reorder needs to be one of [False, True, 'average', 'single', 'complete']"
 Input: "Parameter reorder needs to be one of [False, True, 'average', 'single', 'complete']."
```

Reply:
Ok right. you could maybe use `re.escape("Parameter reorder needs to be one of [False, True, 'average', 'single', 'complete'].")` but if you feel it's not worth the time we just go with the general pattern solution.

Reply:
decided to stringify everything